### DISCUSSION TITLE
Error: Requires VS Code 1.57+ (using 1.61)

### DISCUSSION THREAD

**Discussion by LevitatingBusinessMan** [community/community#7014] [Copilot Conversations] [answered]
Github Copilot is giving an error that my vscode is outdated.

I am using the latest open source build ("Code - OSS" 1.61) from the ArchLinux repositories
https://archlinux.org/packages/community/x86_64/code/

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
Please check this answer: https://github.com/github/feedback/discussions/6629#discussioncomment-1524627
